source/plugin/data/custom/index.rst

```diff
@@ -2,6 +2,35 @@
 Custom Data
 ===========
 
+.. javadoc-import::
+    org.spongepowered.api.data.serialization.DataSerializable
+    org.spongepowered.api.data.manipulator.mutable.DataManipulator
+    org.spongepowered.api.data.DataHolder
+
+Sponge has a powerful :doc:`data <../index>` system, that can do much more than just vanilla features. It's also 
+possible to create your own data objects, allowing you to :doc:`serialize <../serialization>` objects directly to 
+players, entities and more!
+
+To stat making your own data, it's highly recommended that you read up on each of the components of the ecosystem in 
```

Comment:
this could be shortened to "..., we recommend that you ...". We are can use the royal "we" here :)